packages/datatrak-web/src/views/AccountSettingsPage/AccountSettingsSection.tsx

```diff
@@ -27,26 +28,29 @@ const Title = styled(Typography).attrs({
   variant: 'h2',
 })`
   font-size: 1rem;
-  margin-bottom: 0.6rem;
   font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+  margin-block-end: 0.6rem;
 `;
 
 interface AccountSettingsSectionProps {
-  title?: ReactNode;
-  description?: ReactNode;
+  title?: ReactNode | string;
+  description?: ReactNode | string;
+  supplement?: ReactNode;
```

Comment:
Any variable name suggestions? `supplement` doesn’t communicate what this is at all 😕 

This is the secondary/supplementary info in a section that appears below its heading and description